metagraph/src/graph/representation/bitmap/dbg_bitmap.hpp

```diff
@@ -90,6 +90,7 @@ class DBGBitmap : public DeBruijnGraph {
     bool operator==(const DBGBitmap &other) const { return equals(other, false); }
     bool operator!=(const DBGBitmap &other) const { return !(*this == other); }
 
+    using DeBruijnGraph::operator!=;
```

Comment:
Otherwise, we get
```console

/home/adamant/metagraph/metagraph/src/graph/representation/base/sequence_graph.hpp:224:18: error: ‘virtual bool mtg::graph::DeBruijnGraph::operator!=(const mtg::graph::DeBruijnGraph&) const’ was hidden [-Werror=overloaded-virtual=]
  224 |     virtual bool operator!=(const DeBruijnGraph &other) const { return !operator==(other); }
      |                  ^~~~~~~~
/home/adamant/metagraph/metagraph/src/graph/representation/bitmap/dbg_bitmap.hpp:91:10: note:   by ‘bool mtg::graph::DBGBitmap::operator!=(const mtg::graph::DBGBitmap&) const’
   91 |     bool operator!=(const DBGBitmap &other) const { return !(*this == other); }
      |          ^~~~~~~~
cc1plus: all warnings being treated as errors
```

Reply:
We rarely use this function, so I think the best thing would be to implement dummy versions of operator== and operator!= (similar to #481) as the defaults, then override them if necessary. I don't remember why this is implemented this way with `using`.

Reply:
There are already default operators in `DebruijnGraph`. I needed to add `using` here because without it, `virtual bool operator!=(const DeBruijnGraph &other) const` is hidden by `bool operator!=(const DBGBitmap&)`. I think implementing it via `using` should work essentially the same as you suggest, it's just here to make it explicit that default comparison should be used for other graphs instead of `operator!=(const DBGBitmap&)`.

Reply:
ok, then it's not a problem